addons/website/controllers/main.py

```diff
@@ -723,9 +685,9 @@ def theme_customize_bundle_reload(self):
         Reloads asset bundles and returns their unique URLs.
         """
         return {
-            'web.assets_common': request.env['ir.qweb']._get_asset_link_urls('web.assets_common'),
-            'web.assets_frontend': request.env['ir.qweb']._get_asset_link_urls('web.assets_frontend'),
-            'website.assets_editor': request.env['ir.qweb']._get_asset_link_urls('website.assets_editor'),
+            'web.assets_common': request.env['ir.qweb']._get_asset_link_urls('web.assets_common', request.session.debug),
+            'web.assets_frontend': request.env['ir.qweb']._get_asset_link_urls('web.assets_frontend', request.session.debug),
+            'web.assets_backend': request.env['ir.qweb']._get_asset_link_urls('web.assets_backend', request.session.debug),
```

Comment:
> I think assets_editor is not necessary anymore because it's just included in frontend and backend bundles and is no longer it's own package.

You think or you are sure? :wink: I don't understand why there should not be an editor assets which is lazy loaded (instead of increasing the size of both frontend and backend). Although, I guess that could be ok (same as other apps in the end... depends if our files are too much or not compared to the usual custo). The wysiwyg files are not lazy loaded?

> For the backend package, the o_cc values are defined in web_editor.common.scss which is declared in both web.assets_frontend and web.assets_backend. 

Yes, but was not it a mistake that it's declared in common.scss from the beginning? (mistake which can be kept and fixed later).

> When we change the values of o_cc, to have the correct colors in color pickers, we need to reload the backend scss.

To have the right values in the colorpickers, don't you do some rendering inside the iframe then copy the style as discussed? Or do you mean just the values of some variables which are accessed via the backend assets maybe? I think I'd prefer to avoid that, CSS values should be read on the html element of the iframe.